utils/oath-toolkit/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=oath-toolkit
+PKG_VERSION:=2.6.2
+PKG_RELEASE:=1
+PKG_SOURCE:=oath-toolkit-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://download.savannah.gnu.org/releases/oath-toolkit/
+PKG_SOURCE_MD5SUM:=4a05cd4768764843bd5493609a6bdb17
+PKG_LICENSE:=GPL-3.0+
+PKG_LICENSE_FILES:=COPYING
+PKG_MAINTAINER:=Fam Zheng <fam@euphon.net>
+
+include $(INCLUDE_DIR)/package.mk
+
+CONFIGURE_ARGS += \
+	--disable-xmltest \
+	--disable-pskc
+
+define Package/oath-toolkit
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Oath toolkit
```

Comment:
BTW for description section I'll also trim out the last sentence referring to Open Authentication in the name scheme which is not super interesting to end users.